### PR TITLE
Add /sa command

### PR DESCRIPTION
**Pull request by ghost**
/Sa is a shorthand way for /survey answer. It wasn't taken by something else, so I decided that it was the perfect way. It isn't something necessary, but it was a nice, shorthand way to write a command like /warn and /k.

### CODE REVIEW

**Comment by ghost** [comment]
Travis'd and done

File: spacialgaze-plugins/survey.js [hunk 1]
```diff
@@ -161,6 +161,10 @@ function validateAnswer(room, message) {
 }
 
 exports.commands = {
+
```
Comment (HoeenCoder) [inline spacialgaze-plugins/survey.js]: Remove this extra line, its not needed here.

File: spacialgaze-plugins/survey.js [hunk 2]
```diff
@@ -161,6 +161,10 @@ function validateAnswer(room, message) {
...
+	sa: function (target, room, user) {
+		this.parse('/survey answer');
```
Comment (HoeenCoder) [inline spacialgaze-plugins/survey.js]:
```javascript
this.parse('/survey answer ' + target);
```

**Review by HoeenCoder — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Actually done now

File: spacialgaze-plugins/survey.js (hunk 1 above)
Comment (ghost) [reply spacialgaze-plugins/survey.js]: Done

File: spacialgaze-plugins/survey.js (hunk 2 above)
Comment (ghost) [reply spacialgaze-plugins/survey.js]: Also done

**Review by HoeenCoder — APPROVED** [review]

**Pull request merged**